src/hooks/useQuiz.ts

```diff
@@ -52,6 +52,7 @@ const useQuiz: UseQuiz = (quizOptions) => {
         currentQuestion: quizApiState.quizCurrentQuestion,
         results: quizApiState.quizResults,
         selectedOptionsWithAttributes: quizApiState.selectedOptionsWithAttributes,
+        matchedOptions: quizApiState.matchedOptions,
```

Comment:
Can you add documentation around this in `src/stories/Quiz/Hooks/Docs/markdown/StateDocs.md`?

Reply:
[9e749d6](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/148/commits/9e749d6bba3f4e510b6fd98bbe0e260357f63cf4)